styles.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+footer {
+  margin: 0;
+  background-color: #7d3c98;
+  color: white;
+  font-size: 16px;
+  text-align: center;
```

Comment:
Personal preference but use of ‘rem’ here may be better. But I’m sure ‘ox’ works as well

Reply:
Actually hold off on that, we probably should clarify project standard before making this adjustment @MichaelJHTaggart

Reply:
@WesleyMcGhee, project standard is the following design file. Actually this is the task/issue/feature this pull request is meant to resolve: https://github.com/MichaelJHTaggart/wizzi-quills/issues/8

@sponege Could you link your issue to this pull request for Wesley to see exactly the criteria for the pull request. (See the below image on how to do this)

<img width="1297" alt="Screenshot 2023-10-09 at 11 21 18 PM" src="https://github.com/MichaelJHTaggart/wizzi-quills/assets/63270278/8d9a3d58-0778-4558-8fee-0a47e74867eb">